src-electron/sdk/matter.js

```diff
@@ -163,19 +149,23 @@ async function computeStorageImport(
   forcedExternal,
   attributeName
 ) {
-  if (
-    forcedExternal.byName &&
-    forcedExternal.byName[clusterName] &&
-    forcedExternal.byName[clusterName].includes(attributeName)
-  ) {
-    storagePolicy = dbEnum.storagePolicy.attributeAccessInterface
-  }
-  return storagePolicy
+  let updatedStoragePolicy = storagePolicy
+  forcedExternal.some((option) => {
```

Comment:
some is used to check if a condition is met when searching through the array. The purpose of this code seems to be used to update `updatedStoragePolicy` so a foreach and return seems to be the right way to go about this

Reply:
The purpose is to change the policy if the condition is met. No need to do any more work after that.